processor/batchprocessor/Config.pkl.go

```diff
@@ -0,0 +1,16 @@
+// Code generated from Pkl module `BatchProcessorConfig`. DO NOT EDIT.
+package batchprocessor
+
+import "github.com/apple/pkl-go/pkl"
+
+type Config struct {
+	Timeout *pkl.Duration `pkl:"timeout" mapstructure:"timeout"`
```

Comment:
Unfortunately, Pkl uses its own `Duration` type instead of `time.Duration`. I'm not sure what would be a clean way to get `mapstructure` to be able to unmarshal something like "5ms" into a `pkl.Duration`. This is my main blocker right now, since (I suppose) we need `mapstructure` to still be able to work.